src/test/java/redis/clients/jedis/commands/jedis/StreamsCommandsTest.java

```diff
@@ -369,13 +369,11 @@ public void xgroup() {
     StreamEntryID id1 = jedis.xadd("xgroup-stream", (StreamEntryID) null, map);
 
     assertEquals("OK", jedis.xgroupCreate("xgroup-stream", "consumer-group-name", null, false));
-
     assertEquals("OK", jedis.xgroupSetID("xgroup-stream", "consumer-group-name", id1));
-
     assertEquals("OK", jedis.xgroupCreate("xgroup-stream", "consumer-group-name1", StreamEntryID.LAST_ENTRY, false));
 
     jedis.xgroupDestroy("xgroup-stream", "consumer-group-name");
-
+    assertEquals(1L, jedis.xgroupCreateConsumer("xgroup-stream", "consumer-group-name1","myconsumer1"));
```

Comment:
Do this after following assert. Also try to delete that consumer (again). You can also consider using a different groupname.